events/models.py

```diff
@@ -647,7 +661,20 @@ class SuperEventType:
 
     replaced_by = models.ForeignKey('Event', on_delete=models.SET_NULL, related_name='aliases', null=True, blank=True)
 
+    maximum_attendee_capacity = models.PositiveIntegerField(
+        verbose_name=_('maximum attendee capacity'), null=True, blank=True
+    )
+    remaining_attendee_capacity = models.PositiveIntegerField(
```

Comment:
PositiveSmallIntegerField could be used instead: https://docs.djangoproject.com/en/3.2/ref/models/fields/#positivesmallintegerfield